src/commands/createStaticWebApp/tryGetApiLocations.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { RelativePattern, workspace, WorkspaceFolder } from "vscode";
+import { IActionContext } from "vscode-azureextensionui";
+import { telemetryUtils } from '../../utils/telemetryUtils';
+
+const hostFileName: string = 'host.json';
+
+/**
+ * Checks root folder and one level down first, then all levels of tree
+ * If a single function project is found, returns that path.
+ * If multiple projects are found, will prompt
+ * @param workspaceFolder Per the VS Code docs for `findFiles`: It is recommended to pass in a workspace folder if the pattern should match inside the workspace.
+ */
+export async function tryGetApiLocations(context: IActionContext, workspaceFolder: WorkspaceFolder | string): Promise<string[] | undefined> {
+    return await telemetryUtils.runWithDurationTelemetry(context, 'tryGetProject', async () => {
+        const folderPath = typeof workspaceFolder === 'string' ? workspaceFolder : workspaceFolder.uri.fsPath;
+        if (await fse.pathExists(folderPath)) {
```

Comment:
I know the VS Code FileSystem doesn't have an equivalent `pathExists` function, however
if we're planning to support creating a SWA in virtual workspaces we could create our own implementation using the existing VS Code FileSystem api.

Reply:
We actually have this utils file in storage that supports pathExists using VS Code's file system. The plan was to move this into the ui package after we've had some time to test it, so it was my intention to move that now.  I suppose I could do that with this release then

Reply:
Sounds good. We also just merged the wizard loading prompt feature in the ui package so we can get that in this release also.